src/storage/hnsw.cpp

```diff
@@ -63,6 +79,17 @@ xhnsw_add_index (THREAD_ENTRY *thread_p, BTID *btid, int dimension = 10, int hns
 
 int xhnsw_delete_index (THREAD_ENTRY *thread_p, BTID *btid)
 {
+  char filepath[PATH_MAX];
+
+  if (!hnsw_index_directory_created)
+    {
+      if (create_hnsw_index_directory () != NO_ERROR)
+	{
+	  er_log_debug (ARG_FILE_LINE, "Failed to create HNSW Index directory");
```

Comment:
hnsw.cpp 파일에서 추가한 er_log_debug()에서 출력하는 메시지가 디버깅 목적이라면 모두 _er_log_debug()로 일관되게 변경하시는게 좋을 것 같습니다. 추가적으로 발생할 수 없을 것이라고 기대하는 이러한 코드 패스 상에는 assert()를 추가해 주면, 추후 문제 발생 시 빠르게 원인을 파악할 수 있습니다.

```suggestion
	  _er_log_debug (ARG_FILE_LINE, "Failed to create HNSW Index directory");
```